Formula/kore.rb

```diff
@@ -23,14 +23,14 @@ class Kore < Formula
 
   depends_on "pkg-config" => :build
   depends_on macos: :sierra # needs clock_gettime
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   def install
-    ENV.deparallelize { system "make", "PREFIX=#{prefix}", "TASKS=1" }
+    openssl = Formula["openssl@3"]
+    ENV.deparallelize { system "make", "PREFIX=#{prefix}", "OPENSSL_PATH=#{openssl.opt_prefix}", "TASKS=1" }
```

Comment:
I was hoping the CFLAGS would get stored in installed files; however, it looks like `kore` only saves some linker LDFLAGS.